test/unit/annotation_spec.js

```diff
@@ -2209,10 +2208,9 @@ describe("annotation", function () {
     });
 
     it("should compress and save text", async function () {
-      if (isNodeJS && getNodeVersion().major >= 20) {
+      if (isNodeJS) {
         pending(
-          "CompressionStream behaves differently in Node.js >= 20, " +
-            "compared to Firefox, Chrome, and Node.js 18."
+          "CompressionStream behaves differently in Node.js, compared to Firefox and Chrome."
```

Comment:
Could you make an issue to track this problem so we can figure out why this behavior changed all of a sudden? I'm fine with disabling it here, but given that this now impacts all Node.js versions it'd be good to have a tracking issue for it.

Reply:
Done, please see #17399.